agate-core/src/main/java/org/obiba/agate/service/UserGroupSeed.java

```diff
@@ -54,7 +54,7 @@ public void onApplicationEvent(ContextRefreshedEvent contextRefreshedEvent) {
   private void seedConfiguration() {
     Configuration config = configurationService.getConfiguration();
     // convenient setting for having a issuer ID in OAuth responses
-    config.setPublicUrl("http://localhost:8081");
+    config.setPublicUrl(config.hasPublicUrl() ? config.getPublicUrl() : "http://localhost:8081");
```

Comment:
"if not has url then set url" because setting what you get is somehow useless.

Reply:
Why is it useless? The returned value is what the admin had entered via UI which is saved in DB.

Reply:
`config.setPublicUrl(config.getPublicUrl())` is useless

Reply:
Right! :)